File: Lodjinha/app/src/main/java/br/com/bsavoini/lodjinha/api/model/CategoriesResponse.java
package br.com.bsavoini.lodjinha.api.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class CategoriesResponse {
    @SerializedName("data")
    private  List<CategoryModel> categoriesArr;

    public List<CategoryModel> getCategoriesArr() {
        return categoriesArr;
    }
}

File: app/src/main/java/br/com/andrecouto/alodjinha/data/source/remote/service/response/CategoriesResponse.kt
package br.com.andrecouto.alodjinha.data.source.remote.service.response

import br.com.andrecouto.alodjinha.domain.model.lodjinha.Category
import com.google.gson.annotations.SerializedName


data class CategoriesResponse (@SerializedName("data") var data : List<Category>)
